paimon-core/pom.xml

```diff
@@ -284,13 +298,23 @@ under the License.
                             <artifactSet>
                                 <includes combine.children="append">
                                     <include>com.squareup.okhttp3:okhttp</include>
+                                    <include>com.squareup.okio:okio-jvm</include>
```

Comment:
Please modify NOTICE file.

Reply:
I can't find the NOTICE file in `okio-jvm`

Reply:
<img width="247" alt="image" src="https://github.com/user-attachments/assets/e42fffb4-cc2e-485b-8a83-1f2b1ff72978" />

Reply:
ok, I will modify it later.

Reply:
you need add content in /paimon/paimon-core/src/main/resources/META-INF/NOTICE

Reply:
Added